common/src/main/java/com/microsoft/identity/common/internal/controllers/ExceptionAdapter.java

```diff
@@ -211,6 +221,43 @@ public static void applyCliTelemInfo(@Nullable final CliTelemInfo cliTelemInfo,
         }
     }
 
+    private static IntuneAppProtectionPolicyRequiredException getIntuneAppProtectionPolicyRequiredException(@NonNull final TokenErrorResponse errorResponse) {
```

Comment:
I noticed there's a class called MsalBrokerResultAdapter.java, which might essentially be doing the same thing as this class in its exception handling function? (Seems like this Intune exception is being handled there)

Should we centralize the code?